gems/pending/appliance_console/prompts.rb

```diff
@@ -105,14 +105,6 @@ def ask_for_password_or_none(prompt, default = nil)
       ask_for_password(prompt, default).gsub(/^'?NONE'?$/i, "")
     end
 
-    def ask_for_date(prompt)
-      just_ask(prompt, nil, DATE_REGEXP)
-    end
-
-    def ask_for_time(prompt)
-      just_ask(prompt, nil, TIME_REGEXP)
-    end
-
```

Comment:
I think it fits the mold of "ask for something and compare against a regex" that `prompts.rb` seems to define and that we don't really do anywhere else.
Also I don't think that asking for a time or date is uncommon enough to assume that nothing else in the suite of appliance console functions will want to use it in the future.